pool/src/lib.rs

```diff
@@ -446,6 +488,56 @@ impl LendingPoolTrait for LendingPool {
         paused(&env)
     }
 
+    fn get_account_data(env: Env, who: Address) -> Result<(i128, i128, i128), Error> {
+        let AccountData {
+            collateral,
+            debt,
+            npv,
+            liquidation_debt: _,
+        } = Self::calc_account_data(
+            &env,
+            who.clone(),
+            None,
+            &read_user_config(&env, who)?,
+            &read_reserves(&env),
+            false,
+        )?;
+        Ok((collateral, debt, npv))
+    }
+
+    fn liquidate(
+        env: Env,
+        liquidator: Address,
+        who: Address,
+        get_stoken: bool,
+    ) -> Result<(), Error> {
+        liquidator.require_auth();
+        let reserves = read_reserves(&env);
+        let mut user_config = read_user_config(&env, who.clone())?;
+        let account_data =
```

Comment:
We calc the `account_data` 2 times in this method. The only difference is in the `liquidation` parameter.

Is it possible to optimize it somehow (call it only once), since we do many storage calls within it.

Reply:
It is already optimized - `is_good_position` receives `account_data` as `Option<AccountData>` and we pass it as `Some(account_data)`